0004-market-integration-token-standard.md

```diff
@@ -65,38 +80,43 @@ At time of writing, this standard is established with several constraints found
     unimplemented();
   }
 
-  export function getToken(tokenId:string):Token {
+  // returns TokenTypeID, which is also an index
+  export function mintTokenType(name:string, totalSupply:u64):u32 {
     unimplemented();
   }
 
-  export function getTokenOwner(tokenId:string):string {
+  export function getToken(tokenId:u64):Token {
     unimplemented();
   }
 
-  export function getTokenTypes():TokenType[] {
+  export function getTokenType(tokenTypeId:u32) {
     unimplemented();
   }
-  
-  export function getTokensByOwner(ownerId:string, startTokenId:string = null, limit:u32 = 10):Token[] {
+
+  export function getTokenOwner(tokenId:u64):string {
     unimplemented();
   }
 
-  export function getCountByOwner(tokenType: TokenType, ownerId: string) {
+  export function getAllTokenTypes():TokenType[] {
     unimplemented();
   }
-  
-  export function getTokenData(tokenId:string):bytes {
+
+  export function getTokensByOwner(ownerId:string, startTokenId:u64 = 0, limit:u32 = 10):Token[] {
```

Comment:
Is `startTokenId` an index, or an id (an owner may have `ids: [3, 7, 9])`?

Reply:
Added explanations in latest push